packages/cli/src/commands/generate/schema-from-database/generate_schema_command.ts

```diff
@@ -0,0 +1,124 @@
+import { Argv, CommandModule } from 'yargs';
+import { BackendIdentifierResolver } from '../../../backend-identifier/backend_identifier_resolver.js';
+import { ArgumentsKebabCase } from '../../../kebab_case.js';
+import { SecretClient } from '@aws-amplify/backend-secret';
+import { BackendIdentifier } from '@aws-amplify/plugin-types';
+import { SchemaGenerator } from '@aws-amplify/schema-generator';
+
+const DEFAULT_OUTPUT = 'schema.sql.ts';
```

Comment:
should this default to `amplify/data/schema.sql.ts`?

Reply:
Yes, that's more meaningful.